### PR TITLE
chore(windows): build without delphi ✈

### PR DESCRIPTION
**Pull request by mcdurdin**
This makes it possible to run `set NODELPHI=1`, `nmake build` and then only non-Delphi projects will be built.

@keymanapp-test-bot skip

### CODE REVIEW

File: windows/src/test/Makefile [hunk 1]
```diff
@@ -4,7 +4,11 @@
 
 # ----------------------------------------------------------------------
 
+!ifdef NODELPHI
+TARGETS=.virtual
+!else
 TARGETS=test_i3633
+!endif
 CLEAN=test_i3633
```
Comment (ermshiperete) [inline windows/src/test/Makefile:12]: Does CLEAN need to go in the `!else` branch?

**Review by ermshiperete — APPROVED** [review]

File: windows/src/test/Makefile (hunk 1 above)
Comment (mcdurdin) [reply windows/src/test/Makefile:12]: CLEAN doesn't have any Delphi-specific build steps, so it should be okay.

File: docs/build/windows.md [hunk 2]
```diff
@@ -231,6 +231,14 @@ Invoke-WebRequest -Uri https://github.com/rustwasm/wasm-pack/releases/download/v
   Start Delphi IDE once after installation as it will create various environment
   files and take you through required registration.
 
+  * Note: It is possible to build all components that do _not_ require Delphi by
+    adding the environment variable `NODELPHI=1` before starting the build.
+    Currently many components are Delphi-based, but if you are working just in
+    Keyman Core, the compiler, or Keyman Engine's C++ components, you may be
+    able to get away without building them. In this situation, we recommend
+    copying the relevant Delphi-built components into bin folders from a
```
Comment (ermshiperete) [inline docs/build/windows.md:239]: which `bin` folders?

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.166-alpha](https://keyman.com/downloads/releases/alpha/15.0.166)

**Pull request merged**